components/widgets/inputs/input/input.tsx

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import {useIntl} from 'react-intl';
+
 import classNames from 'classnames';
 
+import {ItemStatus} from 'utils/constants';
+
 import './input.scss';
 
 export enum SIZE {
     MEDIUM = 'medium',
     LARGE = 'large',
 }
 
+export type CustomMessageInputType = {type: 'info' | 'error' | 'warning' | 'ok'; value: string} | null;
```

Comment:
Maybe `success` could be more standard than `ok`.

Reply:
agree!